include/seqan3/argument_parser/argument_parser.hpp

```diff
@@ -425,6 +425,62 @@ class argument_parser
         return *sub_parser;
     }
 
+    /*!\brief Checks whether the **short** option identifier (`id`) was set on the command line by the user.
+     * \param[in] id The short option identifier to search for.
+     * \returns `true` if option identifier `id` was set on the command line by the user.
+     *
+     * \details
+     *
+     * You can only ask for option identifiers that were added to the parser beforehand via
+     * seqan3::argument_parser::add_option.
```

Comment:
The API documentation is missing the throw statement and a description when and what this function throws.